src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  margin: 0;
```

Comment:
Set box-sizing: border-box to all the elements with * selector, and don't set margins in * selector.
In the future, we will use reset.css for that. For now just set it for all the classes separately where you need it.